pygraphviz/__init__.py

```diff
@@ -56,6 +56,3 @@
 from .agraph import AGraph, Node, Edge, Attribute, ItemAttribute, DotError
 
 __all__ = ["AGraph", "Node", "Edge", "Attribute", "ItemAttribute", "DotError"]
-
-
-from pygraphviz.tests.test import run as test
```

Comment:
Do we still need this? For now I've removed it because it was imported everywhere.

Reply:
I would suggest that we remove it. It allowed easy syntax for testing from an interpreter after import. But that's not the common way to test and it wasn't well known or advertised (I believe).